manifests/capsule.pp

```diff
@@ -7,11 +7,15 @@
 #
```

Comment:
Then it should not exist? ;-)
But it was quite handy, as it allowed me to test on a plain Katello 3.2 install.